migrations/versions/b765d82110bd_change_metamessages_to_user_level_with_.py

```
@@ -8,6 +8,7 @@
 
 from collections.abc import Sequence
 from typing import Union
+from os import getenv
```

Ruff (0.11.9)

`os.getenv` imported but unused

Remove unused import: `os.getenv`

(F401)